C/manual/tips-appendix.docbook

```diff
@@ -45,6416 +62,6935 @@
     </para>
 
     <sect2 id="fq-currencies">
-      <title>Finance::Quote Sources - Currency source</title>
+      <title>Finance::Quote Sources - Currency Source</title>
```

Comment:
In previous para add a xref, please:
`      ISO currencies. The other two can be selected in the <xref linkend="tool-security-edit"/>.`
This should already have been done years before.

Reply:
done